packages/paste-design-tokens/gulpfile.ts

```diff
@@ -286,7 +286,10 @@ gulp.task('browsersync', () =>
 );
 
 gulp.task('watch', () => {
-  gulp.watch([`${paths.aliases}/**/*.*`, `${paths.globals}/**/*.*`], gulp.series('tokens:all', 'docs'));
+  gulp.watch(
+    [`${paths.aliases}/**/*.*`, `${paths.globals}/**/*.*`, `${paths.tokensDir}/**/*`],
```

Comment:
The watch command wasn't updating the design tokens when I was updating the paste theme's so I fixed it.